nuxeo-features/nuxeo-automation/nuxeo-automation-server/src/main/java/org/nuxeo/ecm/automation/server/jaxrs/batch/Batch.java

```diff
@@ -51,17 +52,61 @@
 
     public static final String CHUNKED_PARAM_NAME = "chunked";
 
+    protected TransientStore transientStore;
+
     protected String key;
 
+    protected String provider;
```

Comment:
`getName` is the registered Handler name. `Provider` is the BlobProvider associated with this Batch

Reply:
OK then this line is confusing: https://github.com/nuxeo/nuxeo/pull/1752/files#diff-d31cfe45bef9d599733d330b91e56893R470, since you use the provider string to fetch the handler:
```
BatchHandler handler = batchManager.getHandlerByName(batch.getProvider());
```

Reply:
Yes, the `getProvider` name is unfortunate, need to think of something better.